code/src/containers/forms/product-form/ProductForm.scss

```diff
@@ -0,0 +1,180 @@
+@import "@design-system";
+
+$gridGap: spacing(5);
+$gridGapHalf: calc($gridGap / 2);
+
+.product-form {
+  display: grid;
+  grid-template-columns: calc(30% - $gridGapHalf) calc(70% - $gridGapHalf);
+  grid-gap: $gridGap;
+  margin-bottom: spacing(5);
+
+  &__header {
+    background-color: var(--new-product-form-head-bg-color, $light-alpha);
+    padding: spacing(5) spacing(8);
+
+    &-main {
+      display: flex;
+      gap: spacing(2);
+      padding: 11.5px spacing(8) !important; // to overwrite mui styles
```

Comment:
Do we need `11.5px`? Can we use `spacing(4)` instead?

Reply:
Thank you for noticing. I needed exact padding there, because header of that section has buttons inside and without exact padding that header would be bigger than other sections' headers. It would be bigger by one pixel but it is still visible because those headers placed one after another.